cmd/device-definitions-api/populate_device_features.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+
+	"github.com/DIMO-Network/device-definitions-api/internal/config"
+	"github.com/DIMO-Network/device-definitions-api/internal/infrastructure/db/models"
+	elastic "github.com/DIMO-Network/device-definitions-api/internal/infrastructure/elasticsearch"
+	elasticModels "github.com/DIMO-Network/device-definitions-api/internal/infrastructure/elasticsearch/models"
+	"github.com/DIMO-Network/shared/db"
+	"github.com/rs/zerolog"
+	"github.com/volatiletech/sqlboiler/v4/boil"
+)
+
+type jsonObj map[string]interface{}
+
+func prepareFeatureData(f map[string]map[string]int) []elasticModels.DeviceIntegrationFeatures {
+	ft := []elasticModels.DeviceIntegrationFeatures{}
+
+	for k, v := range f {
+		var supportLevel int8
+
+		if v["doc_count"] > 0 {
+			supportLevel = 2
```

Comment:
It is hard to know what `2` means here. I think this could be a good use case for an "enum" type (which go doesn't support natively) but you can do something like [this example](https://github.com/DIMO-Network/devices-api/blob/fac677e864105e97cae3315a45ba2e7a64a0a0a0/internal/services/ingest_registrar.go#L98-L107)